packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js

```diff
@@ -134,6 +134,21 @@ export default function({ types: t, template }) {
             noInterop,
           });
 
+          let exportsList = [];
```

Comment:
Some style guides would forbid this, but I felt like it was more readable than my initial implementations using `map` or `reduce` and flattening. Let me know if a change is desired

Reply:
Hmm, I don't find this too hard to read......
```js
const exportsList = [].concat(
  ...Object.values(meta.local).map({ names } => names),
  ...Array.from(meta.source.values(), source => Array.from(source.reexports.keys()))
)
```
...but I'd definitely want comments if written like this.